python/cuml/test/test_random_forest.py

```diff
@@ -539,19 +499,13 @@ def test_rf_classification_sparse(datatype, split_algo, rows_sample,
 @pytest.mark.parametrize('column_info', [unit_param([20, 10]),
                          quality_param([200, 50]),
                          stress_param([400, 100])])
-@pytest.mark.parametrize('rows_sample', [unit_param(1.0), quality_param(0.90),
-                         stress_param(0.95)])
 @pytest.mark.parametrize('datatype', [np.float32])
-@pytest.mark.parametrize('split_algo', [0, 1])
-@pytest.mark.parametrize('max_features', [1.0, 'auto', 'log2', 'sqrt'])
 @pytest.mark.parametrize('fil_sparse_format', ['not_supported', True,
                                                'auto', False])
 @pytest.mark.parametrize('algo', ['auto', 'naive', 'tree_reorg',
                                   'batch_tree_reorg'])
-def test_rf_regression_sparse(datatype, split_algo, mode, column_info,
-                              max_features, rows_sample,
+def test_rf_regression_sparse(datatype, mode, column_info,
```

Comment:
we can use pytest fixtures to create the regression dataset and use it for most if not all of the regression tests.